prow/plugins/dco/dco.go

```diff
@@ -269,7 +269,8 @@ func handle(gc gitHubClient, cp commentPruner, log *logrus.Entry, org, repo stri
 	return takeAction(gc, cp, l, org, repo, pr, commitsMissingDCO, existingStatus, hasYesLabel, hasNoLabel, addComment)
 }
 
-func markdownSHAList(org, repo string, list []github.GitCommit) string {
+// MardkownSHAList prints the list of commits in a markdown-friendly way.
+func MarkdownSHAList(org, repo string, list []github.GitCommit) string {
```

Comment:
one for a follow up, but perhaps there should be a `prow/plugins/internal` package at some point to house this sort of method?

Reply:
> but perhaps there should be a prow/plugins/internal package at some point to house this sort of method?

+1, this would be cleaner. I think there are some more plugins which reuse methods in other plugins.